Sources/ParseSwift/Objects/ParseUser.swift

```diff
@@ -245,7 +245,10 @@ extension ParseUser {
         return API.Command<SignupLoginBody, Self>(method: .POST,
                                                   path: .login,
                                                   body: body) { (data) -> Self in
-            let sessionToken = try ParseCoding.jsonDecoder().decode(LoginSignupResponse.self, from: data).sessionToken
+            guard let sessionToken = try ParseCoding.jsonDecoder().decode(LoginSignupResponse.self, from: data).sessionToken
```

Comment:
This should be `try?` or else it will still throw the original decoding error

Reply:
it will not throw the original decoding error become I set `sessionToken` optional, it will only fail on other parsing errors

Reply:
I believe the server should throw the error and sessionToken should be left in it's original form. I'll take a look soon...

Reply:
I totally agree on that fact @cbaker6 , I did not even think we could have a fix on server side 👍